### ISSUE TITLE
incorrect ownership on /tmp/.ansible halts execution from vagrant

### ISSUE THREAD

**Issue by dj2mn** [robertdebock/ansible-role-mysql#2]
**Describe the bug**
When running this role as part of vagrant provisioning, the ini_file section is creating /tmp/.ansible with owner/group as root:root.

**To Reproduce**
1. Run this role:
2. check ownership on /tmp/.ansible
3. see that it says 0700 root:root instead of 0700 vagrant:vagrant

**Expected behavior**
/tmp/.ansible should be owned by the vagrant user

**Error**
Ansible produces a warning about using the reserved variable name "remote_user" at the beginning of the play but continues anyway, right up until the next step when it tries to access /tmp/.ansible as the vagrant user again.

**Environment**
- Control node OS: OSX 10.14
- Control node Ansible version: 2.7.6
- Managed node OS: CentOS 7

**Additional context**
I was able to work around the problem by changing main.yml to use the template module instead of ini_file.

To be fair this is probably a bug in ini_file itself, but until that's fixed, changing the role to use a template works just fine.

**Comment by robertdebock** [comment]
I'll have a look, thanks for you feedback!

Expect an update somewhere next week.

**Comment by robertdebock** [comment]
Hey, thanks for waiting.

Where does that `remote_user` come from? It's not listed in the role [robertdebock.mysql](https://galaxy.ansible.com/robertdebock/mysql).

I've got a feeling that your playbook, included roles or inventory set that `remote_user`.

I could not reproduce this issue. I've tried `molecule test --scenario-name vagrant` and using my [ansible-tester](https://github.com/robertdebock/ansible-tester) stuff. Both seem to be working.

Let me know if you have more details.

**Comment by dj2mn** [comment]
Hi .. thanks for double checking anyway, I'm mystified where remote_user is coming from as well.. it's definitely not in any of my code (I triple checked that before I logged this bug!) and I just grepped the ini_file.py in my python site-packages, it's not mentioned there either. A real mystery, I'll keep an eye out and if anything shows up that might explain it I'll let you know. Thanks for your time. :)
